src/components/list/Directory.astro

```diff
@@ -0,0 +1,145 @@
+---
+import config from 'virtual:starlight/user-config';
+import {
+	stripLeadingAndTrailingSlashes,
+	ensureTrailingSlash,
+} from 'node_modules/@astrojs/starlight/utils/path';
+import { LinkCard, CardGrid } from '@astrojs/starlight/components';
+import { getCollection } from 'astro:content';
+
+interface Props {
+	/**
+	 * Path relative to /src - e.g "/zh-cn/features"
+	 *
+	 * Note: leading and trailing slashes are dropped "/features/" === "features"
+	 */
+	path: string;
+	/**
+	 *  Filter out pages by keyword - case insensitive
+	 */
+	filterOut?: string[];
+	/**
+	 * Force alphabetical order
+	 */
+	forceAbc?: boolean;
```

Comment:
Could we rename this to `sortAlphabetically` or similar to be a bit more self-descriptive?